src/descriptors/power_system_structs.json

```diff
@@ -1435,9 +1435,9 @@
         },
         {
           "name": "operation_cost",
-          "null_value": "TwoPartCost(nothing)",
-          "data_type": "TwoPartCost",
-          "comment": "Operation Cost of Generation [`TwoPartCost`](@ref)"
+          "null_value": "ThreePartCost(nothing)",
```

Comment:
We should make this a Union of types or abstract on OperationalCost? So we can support also the MarketBid cost.